config/locales/en.yml

```diff
@@ -2492,6 +2492,7 @@ en:
         - your Internet Protocol (IP) address
         - details of which version of <a href='https://www.gov.uk/support/browsers/'>web browser</a> you used
         - information on how you use the site, using <a href='/cookies/'>cookies</a> and page tagging techniques
+      portal_users: If you access the service via the LAA Portal, then we collect your portal login and office account number - check the LAA Portal terms and conditions.
```

Comment:
this needs to be:
`portal_users_html: If you access the service via the LAA Portal, then we collect your portal login and office account number - check the <a href='https://www.gov.uk/government/publications/laa-online-portal-help-and-information#attachment_6742318'>LAA Portal terms and conditions.</a>
`